__tests__/api.test.js

```diff
@@ -468,4 +458,60 @@ describe("News API BACKEND PROJECT", () => {
             })
         })
     })
+    describe("GET api/articles sorting queries", () => {
+        test("default sorting by created date and descending", () => {
+            return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then((response) => {
+                const { body } = response
+                expect(body).toBeSortedBy("created_at", {descending:true})
+            })
+        })
+        test("sorts by passed sort query: topic", () => {
+            return request(app)
+            .get('/api/articles?sort_by=topic')
+            .expect(200)
+            .then((response) => {
+                const { body } = response
+                expect(body).toBeSortedBy("topic", {descending:true})
+            })
+        })
+        test("sorts by passed sort query: votes", () => {
+            return request(app)
+            .get('/api/articles?sort_by=votes')
+            .expect(200)
+            .then((response) => {
+                const { body } = response
+                expect(body).toBeSortedBy("votes", {descending:true})
+            })
+        })
+        test("sorts by passed sort query: votes and chosen order:ascending", () => {
+            return request(app)
+            .get('/api/articles?sort_by=votes&&order=ASC')
+            .expect(200)
+            .then((response) => {
+                const { body } = response
+                expect(body).toBeSortedBy("votes", {ascending:true})
+            })
+        })
```

Comment:
Nice happy path testing, but ideally we should also be able to query for just order on its own and have a test to show this as well. E.g. `/api/articles?order=asc`